ctp2_code/gfx/spritesys/director.cpp

```diff
@@ -3195,7 +3155,7 @@ void DirectorImpl::AddTerminateSound(const Unit &unit)
 	m_actionQueue->AddTail(action);
 }
 
-void DirectorImpl::AddInvokeThroneRoom(void)
```

Comment:
Just a note for the future: Removing these `void`s, should not change anything and is ideally done in a separate commit. This way it can be separated from the other stuff in case something goes wrong there.

Reply:
I totally agree and will in the future do `cleanup` actions in separate commits.